File: API/Features/Billing/DocumentTypes/ViewModels/DocumentTypeListVM.cs
namespace API.Features.Billing.DocumentTypes {

    public class DocumentTypeListVM {

        public int Id { get; set; }
        public string Abbreviation { get; set; }
        public string Description { get; set; }
        public string Batch { get; set; }
        public string LastDate { get; set; }
        public int LastNo { get; set; }
        public bool IsActive { get; set; }
        public string Customers { get; set; }
        public string Suppliers { get; set; }
        public bool IsMyData { get; set; }
        public string Table8_1 { get; set; }
        public string Table8_8 { get; set; }
        public string Table8_9 { get; set; }

    }

}

File: FrontEnd/src/app/features/billing/documentTypes/classes/view-models/documentType-list-vm.ts
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface DocumentTypeListVM {

    id: number
    ship: SimpleEntity
    shipOwner: SimpleEntity
    abbreviation: string
    description: string
    batch: string
    isActive: boolean
    customers: string
    suppliers: string
    isMyData: boolean
    table8_1: string
    table8_8: string
    table8_9: string

}
